core-4/src/main/java/parkinglot/AppConfig.java

```diff
@@ -53,25 +54,12 @@ public Engine defaultV6Engine(){
     }
 
     @Bean("4X4")
-    public Car _4X4(@Qualifier("defaultV6Engine") Engine engine){
-        return new FordBronco(engine);
-    }
-
-    @Bean
-    public Car jurassicCar(@Qualifier("defaultV6Engine") Engine engine){
-        return new FordRaptor(engine);
+    public Car _4X4(){
+        return new FordBronco(defaultV6Engine());
```

Comment:
nope, it's prohibited too, even in the previous task, please, put Engine back to the signature, but don't use any annotations, Spring should understand how to inject defaultV6engine to both cars